ocs_sample_library_preview/SDS/SdsStreamViewMode.py

```diff
@@ -0,0 +1,14 @@
+from enum import Flag
+
+
+class SdsStreamViewMode(Flag):
+    """
+    SdsStreamViewMode 0-32
```

Comment:
should this say "not fully inclusive"?

Reply:
I didn't just because it was a flag and not an enum, but I changed it to be consistent with the enum comments.